apps/web/pages/page-with-gssp.tsx

```diff
@@ -0,0 +1,24 @@
+import {getSession} from "@blitzjs/auth"
+import {GetServerSideProps} from "next"
+import {gSSP} from "../src/server-setup"
+
+export const getServerSideProps: GetServerSideProps = gSSP(async (req, res, ctx) => {
+  const session = await getSession(req, res)
```

Comment:
`session` should be available here using `ctx.session` instead of `getSession`

Reply:
It is! I was just testing `getSession` there